api_test.go

```diff
@@ -1237,14 +1239,6 @@ var _ = Describe("RestHandler", func() {
 				"data": []interface{}{post1JSON},
 			}))
 		})
-
-		It("Extracts multiple parameters correctly", func() {
-			req, err := http.NewRequest("GET", "/v0/posts?sort=title,date", nil)
```

Comment:
hey, we need that. it was very convencient to have these sort query parameters automatically extracted